### PR TITLE
support exporting/replaying the results from graphql queries

### PR DESCRIPTION
**Pull request by nikomatsakis**
r? @chazkiker2

### CODE REVIEW

**Comment by nikomatsakis** [comment]
OK, good enough for now. =)

**Review by chazkiker2 — CHANGES_REQUESTED** [review]
Just the two nits, but otherwise this looks great.

Up to you whether you want to merge this in as-is (after nits) or if you're hoping to bring in whatever other changes you've got going on.

File: src/main.rs [hunk 1]
```diff
@@ -111,10 +122,12 @@ async fn main() {
...
 
+    let graphql = metrics::Graphql::new(&cli.export_graphql, &cli.import_graphql);
+
     if let Some(cmd) = cli.cmd {
         match cmd {
             Cmd::List => {
-                let list_repos = ListReposForOrg::new(org_name, num_days);
+                let list_repos = ListReposForOrg::new(graphql, org_name, num_days);
                 column_names = Some(list_repos.column_names());
                 println!("{:#?}", &list_repos);
```
[the remark this answers is not in the document]
Comment (nikomatsakis) [reply src/main.rs:132]:
```suggestion
```

**Review by chazkiker2 — APPROVED** [review]
Merge whenever 😄

**Pull request merged**